api/v1alpha1/etcd_types.go

```diff
@@ -147,6 +165,10 @@ type BackupSpec struct {
 	// EtcdSnapshotTimeout defines the timeout duration for etcd FullSnapshot operation
 	// +optional
 	EtcdSnapshotTimeout *metav1.Duration `json:"etcdSnapshotTimeout,omitempty"`
+	// OwnerCheck defines parameters related to checking if the seed is an owner
+	// of the shoot. The ownership can change during control plane migration.
+	// +optional
+	OwnerCheck *OwnerCheckSpec `json:"ownerCheck,omitempty"`
```

Comment:
Does it make sense to add `OwnerCheck` to `EtcdSpec` directly? I expect that we'll need this information also when we need the[watchdog principle](https://github.com/gardener/gardener/pull/4638) in the Druid.